Mathlib/Tactic/Positivity/Basic.lean

```diff
@@ -146,3 +167,24 @@ def evalPow : PositivityExt where eval {u α} zα pα e := do
     | .nonnegative pa => ofNonneg pa (← synthInstanceQ (_ : Q(Type u)))
     | .nonzero pa => ofNonzero pa (← synthInstanceQ (_ : Q(Type u)))
     | .none => pure .none
+
+/-- Port note: in mathlib3 this alias is `private`, but mathlib4's `alias` does not
+(yet) support that. -/
+alias abs_pos ↔ _ abs_pos_of_ne_zero
+
+/-- The `positivity` extension which identifies expressions of the form `|a|`. -/
+@[positivity |(_ : α)|, Abs.abs _]
```

Comment:
These two patterns are the same. The reason add and mul have two patterns instead of one  is because they match both `HAdd.hAdd` and `Add.add`.

Reply:
Done, and similar for `evalInv`.